### ISSUE TITLE
database hostname=db hardcoded in klaros image

### ISSUE THREAD

**Issue by x3nb63** [klaros-testmanagement/klaros-docker#46]
[PostgreSQL/klaros/files/catalina-wrapper.sh](https://github.com/klaros-testmanagement/klaros-docker/blob/e36e5826393fadb1644fc9557fa6af5818461cd1/PostgreSQL/klaros/files/catalina-wrapper.sh) contains this line:

```
echo "hibernate.connection.url = jdbc:postgresql://db/${DATABASE_NAME}"
```

that `db` is actually the hostname, which klaros will use to connect to the database. This just happens to work since `docker-compose.yml` spins a container with that name.

However, its no good if the image is used by some other orchestrator, Kubernetes in my case, and/or when the database is to reside on some generic postgresql server, ie. a database cluster or so.

I suggest adding a `DATABASE_HOST` environment variable with default value `db`  to the klaros Dockerfile. Then its possible to specify something else.

**Comment by stolp** [comment]
This sounds like a useful enhancement. Do you care to submit a pull request for this?

**Comment by x3nb63** [comment]
very busy at the moment

can you do it along with some fix going to be released anyway?

**Comment by stolp** [comment]
Thanks for responding.

We are likewise busy to get version 5.1.0 out the door later this month. I may fit this in with the 5.1.0 Docker release afterwards.

**Comment by stolp** [comment]
Resolved via #49.
